src/modules/acbs/acbs-facility-loan-transaction.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpService } from '@nestjs/axios';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import AcbsConfig from '@ukef/config/acbs.config';
+
+import { AcbsHttpService } from './acbs-http.service';
+import { AcbsGetFacilityLoanTransactionResponseItem } from './dto/acbs-get-facility-loan-transaction-response.dto';
+import { getLoanTransactionNotFoundKnownAcbsError } from './known-errors';
+import { createWrapAcbsHttpGetErrorCallback } from './wrap-acbs-http-error-callback';
+
+@Injectable()
+export class AcbsFacilityLoanTransactionService {
+  private readonly acbsHttpService: AcbsHttpService;
+
+  constructor(
+    @Inject(AcbsConfig.KEY)
+    config: Pick<ConfigType<typeof AcbsConfig>, 'baseUrl'>,
+    httpService: HttpService,
+  ) {
+    this.acbsHttpService = new AcbsHttpService(config, httpService);
+  }
+
+  async getLoanTransactionByBundleIdentifier(bundleIdentifier: string, idToken: string): Promise<AcbsGetFacilityLoanTransactionResponseItem> {
+    const { data: loanTransaction } = await this.acbsHttpService.get<AcbsGetFacilityLoanTransactionResponseItem>({
+      path: `/BundleInformation/${bundleIdentifier}?returnItems=true`,
+      idToken,
+      onError: createWrapAcbsHttpGetErrorCallback({
+        messageForUnknownError: `Failed to get the loan transaction with bundle identifier ${bundleIdentifier}.`,
+        knownErrors: [getLoanTransactionNotFoundKnownAcbsError(bundleIdentifier)],
+      }),
+    });
+    return loanTransaction;
+  }
+}
```

Comment:
Ah, this is interesting. I thought this would be a `AcbsBundleInformationService` instead (in fact, I think Audrius just merged one of these into main) because the `acbs` module should function like a client library for ACBS - we'll be able to re-use this a lot more that way.

The interesting question if we did that would be "how do we convert the _BundleInformation not found_ error into a _Loan transaction not found_ error?". I think the most sensible way to do that is just a try-catch in the service.

We're going to need to solve this when we re-use this code for other endpoints, so maybe for now we should just:
- leave the error handling as it is
- move this method into the AcbsBundleInformationService that Audrius has made

Reply:
Moved this method into `AcbsBundleInformationService` and left as is for now.